### DISCUSSION TITLE
What is the maximum amount of correct answers a question can have?

### DISCUSSION THREAD

**Discussion by jherskow** [Schweinepriester/github-profile-achievements#225] [Q&A] [answered]
What is the maximum amount of correct answers a question can have?

**Comment by mezotv** [comment]
According to my tests its only 1 correct answer per q&a

**Comment by trentmillar** [comment]
🚜 🚚 🚗 🚓 🚌 🚒

**Comment by Chameleon-Coder** [comment, marked as answer]
One. I speak from experience. Please mark me as correct.

**Comment by thenithinbalaji** [comment]
One. 
But you can still unmark an answer and mark it again.

**Comment by abdorizak** [comment]
1
